api/config/module/feedback/v1/feedback.proto

```diff
@@ -20,14 +20,16 @@ message Survey {
   RatingOptions rating_options = 3 [ (validate.rules).message = {required : true} ];
 }
 
-message SurveyOrigin {
-  oneof type {
-    option (validate.required) = true;
+enum Origin {
```

Comment:
wanted to use the same enum defined here https://github.com/lyft/clutch/blob/main/api/feedback/v1/feedback.proto#L21-L26 but wasn't able to import it so I tried the method here https://stackoverflow.com/questions/8553147/is-it-possible-to-use-external-enums-outside-message-definitions-with-protocol which also didn't work. Example of what I tried for the latter approach: I had this import defined at the top of the file `import "/feedback/v1/enums.proto";` and then imported the enum like so `clutch.feedback.v1.<enum name>` but got undefined errors. am I missing something? Also, is it good practice to try to reuse the enum?